builtin/credential/ldap/path_config_rotate_root_test.go

```diff
@@ -0,0 +1,60 @@
+package ldap
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	logicaltest "github.com/hashicorp/vault/helper/testhelpers/logical"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+// This test relies on an external ldap server with a suitable person object (cn=admin,dc=planetexpress,dc=com)
+// with bindpassword "admin". - see the backend_test for more details.
+// This test will not run unless VAULT_ACC is set to something
+func TestRotateRoot(t *testing.T) {
```

Comment:
nit: mainly curious to see if following style precedents for other rotate root tests would help here: since this is an acceptance test, would it be worth making this test more dynamic by taking in the `BindDN` and `BindPass` fields from environment variables? We can skip the test if the vars are not set

Reply:
oh that's a good call - i had just checked on the other auth/ldap tests, but that's a good point

Reply:
Is there any reason we aren't using

https://github.com/hashicorp/vault/blob/88a791a42bec2b77151fe95578d46442282e4bf9/helper/testhelpers/ldap/ldaphelper.go#L19

like the other tests in `builtin/credential/ldap/backend_test.go`? I think that should allow it to run in CI and local tests without the need to manually spin up a container.

Reply:
That make sense - to be honest I just wasn't sure if it was appropriate for a different test case, but since i explicitly tried to maintain the same preconditions, I can probably just use it.

Reply:
+1 on a container based approach so it can be ran multiple times, and not affect other tests dependent on the original `bindpass`

Reply:
This should be done now!